### PR TITLE
Am footers

### PR DESCRIPTION
**Pull request by Nifargo**
New selectors for the "Footers" menu item
New test data for the "Footers" menu item
New quick-autotest for the "Footers" menu item with the US https://trello.com/c/aZNujjOd

### CODE REVIEW

**Comment by Burdishka** [comment]
Before push your branch always make sure that you have all updates from master. You need to update your branch!

**Comment by Nifargo** [comment]
> Before push your branch always make sure that you have all updates from master. You need to update your branch!

yes, I forgot about it and now I have a problem with it. What should I do?

File: data/testData.json [hunk 1]
```diff
@@ -28,7 +28,10 @@
     "integer": "2",
     "decimal": "2.20"
   },
-  "arithmeticInline": {
+  "footers": {
+    "text": "25021987"
+  },
+"arithmeticInline": {
```
Comment (SergeiKuznetsov) [inline data/testData.json:34]:
```suggestion
  "arithmeticInline": {
```

Comment (SergeiKuznetsov) [reply data/testData.json:34]: Зачем изменил эту строку?

**Review by SergeiKuznetsov — CHANGES_REQUESTED** [review]

File: data/selectors.json [hunk 2]
```diff
@@ -132,30 +132,31 @@
...
+ "maxFtrs": "//tbody/tr[@id='add-row-72']/td[1]/button[1]",
+ "maxFtrsInt": "//textarea[@id='t-72-r-1-int']",
+ "maxFtrsIntDis": "//textarea[@id='f-72-int']",
+ "countFtrs": "//tbody/tr[@id='add-row-73']/td[1]/button[1]",
+ "countFtrsInt": "//textarea[@id='t-73-r-1-int']",
+ "averageFtrs": "//tbody/tr[@id='add-row-74']/td[1]/button[1]",
+ "averageFtrsInt": "//textarea[@id='t-74-r-1-int']",
+ "averageFtrsIntDis": "//textarea[@id='f-74-int']",
+ "saveBtn": "//button[@id='pa-entity-form-save-btn']"
+},
```
Comment (SergeiKuznetsov) [inline data/selectors.json:159]:
```suggestion
    "footerTitle": "//b[contains(text(),'Footers')]",
    "sideBar": "//div[@class='sidebar']",
    "footerMenu": "//p[contains(text(),'Footers')]",
    "footerNew": "//i[contains(text(),'create_new_folder')]",
    "sumControl": "//input[@id='sum_control']",
    "minControl": "//input[@id='min_control']",
    "countControl": "//input[@id='count_control']",
    "averageControl": "//input[@id='average_control']",
    "sumFts": "//tbody/tr[@id='add-row-70']/td[1]/button[1]",
    "int1": "//textarea[@id='t-70-r-1-int']",
    "intDis": "//textarea[@id='f-70-int']",
    "intControl": "//textarea[@id='t-70-r-1-control']",
    "minFtrs": "//tbody/tr[@id='add-row-71']/td[1]/button[1]",
    "minFtrsInt": "//textarea[@id='t-71-r-1-int']",
    "minFtrsIntDis": "//textarea[@id='f-71-int']",
    "maxFtrs": "//tbody/tr[@id='add-row-72']/td[1]/button[1]",
    "maxFtrsInt": "//textarea[@id='t-72-r-1-int']",
    "maxFtrsIntDis": "//textarea[@id='f-72-int']",
    "countFtrs": "//tbody/tr[@id='add-row-73']/td[1]/button[1]",
    "countFtrsInt": "//textarea[@id='t-73-r-1-int']",
    "averageFtrs": "//tbody/tr[@id='add-row-74']/td[1]/button[1]",
    "averageFtrsInt": "//textarea[@id='t-74-r-1-int']",
    "averageFtrsIntDis": "//textarea[@id='f-74-int']",
    "saveBtn": "//button[@id='pa-entity-form-save-btn']"
  },
```

**Review by SergeiKuznetsov — CHANGES_REQUESTED** [review]

**Review by SergeiKuznetsov — APPROVED** [review]

**Pull request merged**